src/components/Time/LocalTime.tsx

```diff
@@ -1,27 +1,69 @@
 import * as React from 'react';
 import { Tooltip } from '@patternfly/react-core';
 import { KialiIcon } from '../../config/KialiIcon';
+import { TimeInMilliseconds } from 'types/Common';
 
 interface TimeProps {
   time: string;
 }
 
+const defaultOptions = {
+  month: 'short',
+  day: 'numeric',
+  hour: '2-digit',
+  minute: '2-digit'
+} as any;
+
+export const toString = (time: TimeInMilliseconds, options?: any): string => {
```

Comment:
Changes look good, but I think these exported funcs would better be in a utility file rather than staying in component

Reply:
Makes sense, I'll create Time/utils.tsx and move to there.